src/App.js

```diff
@@ -1,8 +1,23 @@
 import React from 'react'
-
+import {Homepage} from './pages/Homepage/Homepage'
+import { ProductListing } from './pages/ProductListing/ProductListing'
+import { Cart } from './pages/Cart/Cart'
+import { Wishlist } from './pages/Wishlist/Wishlist'
+import { Login } from './pages/Login/Login'
+import { Signup } from './pages/Signup/Signup'
```

Comment:
You can make in index file in pages folder and export all the components from there and import it in here from index.js only

Reply:
done